messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistView.kt

```diff
@@ -46,119 +50,120 @@ class GistView @JvmOverloads constructor(
     }
 
     fun setup(message: Message) {
-        inAppMessagingManager.dispatch(InAppMessagingAction.SetupGistView(message))
+        logger.debug("GistView setup: $message")
         currentMessage = message
         currentMessage?.let { message ->
             val engineWebConfiguration = EngineWebConfiguration(
-                siteId = GistSdk.getInstance().siteId,
-                dataCenter = GistSdk.getInstance().dataCenter,
+                siteId = store.siteId,
+                dataCenter = store.dataCenter,
                 messageId = message.messageId,
                 instanceId = message.instanceId,
-                endpoint = GistSdk.gistEnvironment.getEngineApiUrl(),
+                endpoint = store.environment.getEngineApiUrl(),
                 properties = message.properties
             )
             engineWebView?.setup(engineWebConfiguration)
         }
     }
 
     fun stopLoading() {
-        inAppMessagingManager.dispatch(InAppMessagingAction.EngineStopLoading)
```

Comment:
This is dismissed state. I was asking for cancelled state, when the message is cancelled and dismissed because route has changed and we need to stop further loading and sending callbacks to customer.